tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,150 @@
+| AuditEventUpdateUserProfile |  |  |  |
+| AuditEventUpdateMFA | enableMFA | bool |Enable / disable MFA |
+| AuditEventCreateProject | projectName | string | Project name |
+| AuditEventUpdateProject | oldProjectName <br/> newProjectName | string <br/> string | Old Project name <br/> New project name |
+| AuditEventDeleteProject | projectName | string | Project name |
+| AuditEventInviteUserIntoProject | email <br/> Role | string <br/> string | Email <br/> Role name |
+| AuditEventDeleteProjectUser | email <br/> Role | string <br/> string | Email <br/> Role name |
+| AuditEventUpdateOrg | orgName <br/> timeZone | string <br/> uint| Organization name <br/> Time zone |
+| AuditEventCreateIntegration | integrationType | string | Integration type |
+| AuditEventDeleteIntegration | integrationType | string | Integration type |
+| AuditEventListOrgUsers |  |  |  |
+| AuditEventListProjectUsers |  |  |  |
+| AuditEventAddNewPaymentMethod | cardNumber | string | Payment card number |
+| AuditEventUpdatePaymentMethod | cardNumber | string | Payment card number |
+| AuditEventDeletePaymentMethod |  |  |  |
+| AuditEventCreateAWSVpcPeering |  |  |  |
+| AuditEventCreateGCPVpcPeering |  |  |  |
+| AuditEventListAWSVpcPeering |  |  |  |
+| AuditEventListGCPVpcPeering |  |  |  |
+| AuditEventDeleteAWSVpcPeering |  |  |  |
+| AuditEventDeleteGCPVpcPeering |  |  |  |
+| AuditEventGetProjectTrafficFilter |  |  |  |
+| AuditEventUpdateProjectTrafficFilter |  |  |  |
+| AuditEventGetTrafficFilter |  |  |  |
+| AuditEventUpdateTrafficFilter |  |  |  |
+| AuditEventCreateProjectCIDR |  |  |  |
+| AuditEventGetProjectCIDR |  |  |  |
+| AuditEventGetProjectRegionCIDR |  |  |  |
+| AuditEventDeleteBackupInRecycleBin |  |  |  |
+| AuditEventChangeClusterRootPassword |  |  |  |
+| AuditEventCreateImportTask |  |  |  |
+| AuditEventCancleImportTask |  |  |  |
+| AuditEventExitImportTask |  |  |  |
+| AuditEventCreateCluster | clusterName | string | Cluster name |
+| AuditEventDeleteCluster | clusterName | string | Cluster name |
+| AuditEventScaleCluster |  |  |  |
+| AuditEventCreateBackup | backupName | string | Backup name |
+| AuditEventDeleteBackup |  |  |  |
+| AuditEventRestoreBackup | clusterName | string | Cluster name |
+| AuditEventUpdateAuditLogStatus | enableAuditLog | bool | Enable / disable SQL auditing |
+| AuditEventCreateAuditLogAccessRecord |  |  |  |
+| AuditEventDeleteAuditLogAccessRecord |  |  |  |
+| AuditEventUpdateUserRole | oldRole <br/> newRole | string <br/> string | Old role name <br/> New role name |
+
+> **Note:**
+>
+> The supplementary description of the field information is as follows:
+> - AuditEventAddNewPaymentMethod: The field cardNumber (payment card number) has been desensitized.
+> - AuditEventUpdatePaymentMethod: The field cardNumber (payment card number) has been desensitized, and the UserID is not recorded in audit log temporarily.
+> - AuditEventDeletePaymentMethod: UserID is not recorded in audit log temporarily.
```

Comment:
I think appending this information to the corresponding rows of the above table might look better.

Reply:
The description in the table is ultimately a complete function introduction, and should not contain special instructions. It is more reasonable to prompt information for special situations.